io-engine/src/lvs/mod.rs

```diff
@@ -192,8 +192,13 @@ impl IPoolProps for Lvs {
             .crypto_base_bdev()
             .map(Bdev::new)
             .unwrap_or_else(|| self.base_bdev());
-
-        vec![disk_bdev.bdev_uri_str().unwrap_or_else(|| "".into())]
+        let bdev_uri = disk_bdev.bdev_uri_str().unwrap_or_else(|| "".into());
+        let bdev_path = if bdev_uri.starts_with("malloc") {
+            bdev_uri.split("&").next().unwrap_or("")
+        } else {
+            bdev_uri.split("?").next().unwrap_or("")
+        };
+        vec![bdev_path.to_string()]
```

Comment:
We should not remove all parameters, they may have important information.
We can just use the original:

```suggestion
        vec![disk_bdev.bdev_uri_original().map(|u| u.to_string()).unwrap_or_default()]
```

Reply:
We wanted to remove the bdev uuid from the disk path right? We are removing the `uuid=<bdev_id>`. 

If we need this, We can have it as a separate field perhaps?